ok_tools/templates/home.html

```diff
@@ -7,6 +7,22 @@
 {% block content %}
 {% if user.is_authenticated %}
   <p>Hi {{ user.email }}!</p>
+
+  {% if not user.profile.verified %}
+  <p>
+    {% blocktranslate %}
+    This account is not verified. Please print the application form, sign it and send it back to us. The account will then be verified.
```

Comment:
I think in addition to somehow sending the application form, the personal ID needs to be shown to an employee to verify the profile.

Reply:
I have changed the wording to reflect this.